storage/innobase/xtrabackup/utils/debian/control

```diff
@@ -17,8 +17,8 @@ Build-Depends: automake,
                libev-dev,
                libcurl-dev,
                lsb-release,
-               python-sphinx (>= 1.0.1),
-               python-docutils (>= 0.6)
+               python-sphinx (>= 1.0.1) | python3-sphinx (>= 1.8.5),
+               python-docutils (>= 0.6) | python3-docutils (>= 0.16)
```

Comment:
are you sure that minimal needed versions for python3* are correct?

Reply:
It's the only available versions:
```
root@ubuntu-20:/home/vagrant# apt policy python3-sphinx
python3-sphinx:
  Installed: (none)
  Candidate: 1.8.5-7ubuntu3
  Version table:
     1.8.5-7ubuntu3 500
        500 http://archive.ubuntu.com/ubuntu focal/main amd64 Packages
        500 http://archive.ubuntu.com/ubuntu focal/main i386 Packages

root@ubuntu-20:/home/vagrant# apt policy python3-docutils
python3-docutils:
  Installed: (none)
  Candidate: 0.16+dfsg-2
  Version table:
     0.16+dfsg-2 500
        500 http://archive.ubuntu.com/ubuntu focal/main amd64 Packages
        500 http://archive.ubuntu.com/ubuntu focal/main i386 Packages
```